contrib/cf/pcf-tile/tile.yml

```diff
@@ -0,0 +1,158 @@
+---
+# The high-level description of your tile.
+# Replace these properties with real values.
+#
+name: open-service-broker-azure
+icon_file: resources/icon.png
+label: Microsoft Open Service Broker of Azure
```

Comment:
This should be Microsoft Open Service Broker *for* Azure, not *of* Azure.

Reply:
Thanks! Fixed.